.github/ISSUE_TEMPLATE.md

```diff
@@ -12,21 +12,19 @@
 
 #### Sonata packages
 
-```bash
-$ composer show sonata-project/*
-# Put the result here.
+```
+$ composer show --latest 'sonata-project/*'
```

Comment:
This is not normal.

Reply:
This is not a part of my changes. Maybe this is a result of `rebase`? I did something wrong?